src/main/java/org/folio/processor/RuleProcessor.java

```diff
@@ -83,6 +86,7 @@ public String process(EntityReader reader, RecordWriter writer, ReferenceDataWra
    * @return the list of the generated VariableField of marc record
    */
   public List<VariableField> processFields(EntityReader reader, RecordWriter writer, ReferenceDataWrapper referenceData, List<Rule> rules, ErrorHandler errorHandler) {
+    log.info("processFields:: the rules param: {}",rules);
```

Comment:
Rules object is list. I assume we have uninformative hash code here.

Reply:
@DmytroBykov1 
1) Why do we need to use default Lombok representation for json object? Typically, we make print logs for unchanged json.
2) You made logging of rule in the several places. Do you expect that rule will be changed during business flow?

Reply:
Will Lombok annotation be applied to JDK List?